qcodes/tests/drivers/test_keysight_b220x.py

```diff
@@ -0,0 +1,216 @@
+import pytest
+from pyvisa.errors import VisaIOError
+from qcodes.instrument_drivers.Keysight.keysight_b220x import KeysightB220X
+
+@pytest.fixture
+def uut() -> KeysightB220X:
+    try:
+        uut = KeysightB220X('switch_matrix',
+                            address='GPIB::22::INSTR')
```

Comment:
the test will not work if the address is incorrect. Shall we put a "garbage" address here so that the test will not get accidentally executed? and perhaps a nice error message that the user should enter a correct address here in order to run the tests on the real instrument?

Reply:
Done. changed address to `'insert_Keysight_B2200_VISA_resource_name_here'`